### DISCUSSION TITLE
Bought the cheats

### DISCUSSION THREAD

**Discussion by irhrhd** [Prodigy-Hacking/ProdigyMathGameHacking#613] [(A) Anything]
Some kid i know bought the dam cheat menu for 30 bucks And i told him he needs a refund cause the menu is free and he tried but they wouldn’t give it back XD

**Comment by PatheticMustan** [comment]
sad, would you mind telling us who was selling it?

**Reply by irhrhd** [reply]
I dont really know it just is a pastebin where they link their paypal to buy it and their paypal name is just random numbers

**Reply by irhrhd** [reply]
I dont have the pastebin the skid told me that

**Reply by ghost** [reply]
o wtf

**Reply by ghost** [reply]
tell him to contact paypal
